File: Ejercicio Pdf/Semana3/Actividad Semanal/C#/Ejercicio_ActividadSemanal_1/Ejercicio_ActividadSemanal_1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio_ActividadSemanal_1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Calcular el área y el perímetro de un terreno rectangular.
            Console.WriteLine("Ingrese la Base del terreno rectangular:");
            double baserec = double.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese la altura del terreno rectangular");
            double altura = double.Parse(Console.ReadLine());
            double area = baserec * altura;
            double perimetro = 2 * (baserec + altura);
            Console.WriteLine("El Area del terreno rectangular es de: " + area);
            Console.WriteLine("El perimetro del terreno rectangular es de: " + perimetro);
        }
    }
}

File: Ejercicio Pdf/Semana3/Actividad Semanal/Python/Ejercicio_ActividadSemanal_1.py
#Calcular el área y el perímetro de un terreno rectangular.

print("Calcular el área y el perímetro de un terreno rectangular.")
area = 0
perimetro = 0

base = float(input("Ingrese la base del terreno rectangular: "))
altura = float(input("Ingrese la altura del terreno rectangular: "))
area = base * altura
perimetro = 2 * (base + altura)
print("El área del terreno rectangular es: ", area)
print("El perímetro del terreno rectangular es: ", perimetro)


